activesupport/lib/active_support/message_verifier.rb

```diff
@@ -24,6 +24,12 @@ module ActiveSupport
   # hash upon initialization:
   #
   #   @verifier = ActiveSupport::MessageVerifier.new('s3Krit', serializer: YAML)
+  #
+  # +MessageVerifier+ creates HMAC signatures using SHA1 hash algorithm by default.
```

Comment:
I'm not a native speaker but I don't think we can say a SHA1. By SHA1 I mean the algorithm itself, not it's implementation in Ruby OpenSSL library or another language or library. This is the same thing we can't say a Jon Moss :smile: